components/dirigeants-section/beneficiaires.tsx

```diff
@@ -4,19 +4,19 @@ import InpiPartiallyDownWarning from '#components-ui/alerts/inpi-partially-down'
 import { HorizontalSeparator } from '#components-ui/horizontal-separator';
 import { INPI } from '#components/administrations';
 import { Section } from '#components/section';
+import { LoadingSection } from '#components/section/loading';
 import { FullTable } from '#components/table/full';
 import { EAdministration } from '#models/administrations';
 import {
   IAPINotRespondingError,
   isAPINotResponding,
 } from '#models/api-not-responding';
 import { IBeneficiaire, IImmatriculationRNE } from '#models/immatriculation';
-import { formatDate, formatDatePartial } from '#utils/helpers';
-import { Siren } from '#utils/helpers';
+import { Siren, formatDatePartial } from '#utils/helpers';
 import AdministrationNotResponding from '../administration-not-responding';
 
 type IProps = {
-  immatriculationRNE: IImmatriculationRNE | IAPINotRespondingError;
+  immatriculationRNE: IImmatriculationRNE | IAPINotRespondingError | null;
```

Comment:
How could we use a cleaner way to materialize the loading state ? 

For instance use a dedicated type like `IAPILoading`rather than `null` with its dedicated type check `isAPILoading` ?